references/index_new_fields.md

```diff
@@ -0,0 +1,103 @@
+# Accept new fields
+
+`AcceptNewFields` determines what MeiliSearch should do with new fields found during documents addition.
+
+When `AcceptNewFields` is set to **true** (*default*), every new field will be added to the [searchable-attributes](/references/searchable_attributes.md) and the [displayed-attributes](/references/displayed_attributes.md) list.<br>
+When `AcceptNewFields` set to **false**, they will be stored but neither searchable or displayed on the returned documents.
+
+::: tip
+
+When `AcceptNewFields` is false, the fields are still stored. This means you can add them to the [searchable attributes](/references/searchable_attributes.md) list or the [displayed attributes](/references/displayed_attributes.md) list at any time.
+
+:::
+
+Child route of the [settings route](/references/settings.md).
+
+Stop-words can also be updated directly through the [add settings route](/references/settings.md#add-settings) at the same time than the other settings.
+
+## Get accept new fields
```

Comment:
Maybe is it better to do titles like `Get acceptNewFields` instead of `Get accept new fields`? Because it looks like a "real" sentence without any sense.

Reply:
fixed by #184